tests/test_prep_run_nvnmd.py

```
@@ -0,0 +1,307 @@
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)